CMakeLists.txt

```diff
@@ -49,8 +49,8 @@ endif()
 option(USE_AVX2 "enable avx2 code" OFF)
 option(USE_NETNS "enable networking namespace support. Linux only" OFF)
 option(NATIVE_BUILD "optimise for host system and FPU" ON)
-option(EMBEDDED_CFG "optimise for older hardware or embedded systems" OFF)
-option(BUILD_LIBLOKINET "build liblokinet.so" ON)
+option(BUILD_LIBLOKINET "same as WITH_EMBEDDED_LOKINET" ON)
```

Comment:
Why keep this?

Reply:
so i break less things in CI. (lazy)

Reply:
I feel like this will come back to bite us in the ass, let's nuke the old name.

Reply:
yea, okay